boards/inkplate6color.json

```diff
@@ -0,0 +1,31 @@
+{
+    "build": {
+        "arduino": {
+            "ldscript": "esp32_out.ld"
+        },
+        "core": "esp32",
+        "extra_flags": "-DARDUINO_INKPLATE6COLOR -DBOARD_HAS_PSRAM -mfix-esp32-psram-cache-issue",
```

Comment:
Is `-DARDUINO_INKPLATE6COLOR` correct, or should it be `-DARDUINO_INKPLATECOLOR` (without the `6`)?

The Arduino examples all check for the define without the `6`. 

https://github.com/SolderedElectronics/Inkplate-Arduino-library/blob/3f1908e3cda550b156e3e5b8adeb41177a456f9e/examples/Inkplate6COLOR/Basic/Inkplate6COLOR_Full_Screen_Colors/Inkplate6COLOR_Full_Screen_Colors.ino#L16

I also can't find any reference to the define with the `6` in Inkplate-Arduino-Library, whereas it references the one without the `6` a bunch.

https://github.com/SolderedElectronics/Inkplate-Arduino-library/blob/3f1908e3cda550b156e3e5b8adeb41177a456f9e/src/Inkplate.h#L124